### DISCUSSION TITLE
What's with all the recent undocumented releases?

### DISCUSSION THREAD

**Discussion by abusch** [awslabs/aws-sdk-rust#912] [General]
Hi, I've noticed there's been 4 or 5 new entries recently in the [changelog](https://github.com/awslabs/aws-sdk-rust/blob/main/CHANGELOG.md) with no mention of what has changed. Just curious if that information has moved somewhere else?

**Comment by jdisanti** [comment]
We are exercising our release infrastructure by trying to update our SDK models daily, as a step towards doing daily releases like the other SDKs are. Unfortunately, the changelog part of that isn't quite working yet, so it's showing up as empty release notes. Sorry for any thrash or confusion this may be causing. We should be finished with this exercise soon.

**Reply by abusch** [reply]
Thanks for the clarification!

**Reply by syphar** [reply]
@jdisanti 
I'm maintaining docs.rs and stumbled onto this discussion. 

A daily release of all ~380 crates would be tricky to handle in the setup we have right now. 
If this would start soon, we probably would deprioritize the builds even more. 

We _are_ working on setting up a second builder process, and also [will try to start caching build artifacts in some cases](https://github.com/rust-lang/docs.rs/pull/2079). Any help is appreciated. 
It would be awesome if starting the daily releases could be coordinated with us (I started a discussion [on zulip](https://rust-lang.zulipchat.com/#narrow/stream/356853-t-docs-rs/topic/workspace.20artifact.20caching.20-.3E.20AWS.20SDKs)).

**Reply by Velfi** [reply]
Luckily, we don't expect to release that many crates every day. It'll likely be 5-8 crates a day. We will be releasing all of our crates on one day once we go to 1.0, and we'll also have to do that any time we update a runtime crate. I think we're going to try to avoid that by setting the versions for the SDK crates such that they'll automatically pick up the new runtime crate versions.

**Reply by ysaito1001** [reply]
For reference, in the latest three releases as of today,
- [release-2023-10-09](https://github.com/awslabs/aws-sdk-rust/releases/tag/release-2023-10-09) published 21 crates
- [release-2023-10-06](https://github.com/awslabs/aws-sdk-rust/releases/tag/release-2023-10-06) published 356 crates
- [release-2023-10-05](https://github.com/awslabs/aws-sdk-rust/releases/tag/release-2023-10-05) published 2 crates

**Reply by syphar** [reply]
@jdisanti seeing that the most recent release has a changelog entry again, 

is there any way we could get a changelog for the other releases that lack the information? 

I would like to prevent having to go through the diff. 

Perhaps some middle ground, so looking at the PRs that were released with each release?

**Reply by jdisanti** [reply]
The releases without changelog entries were purely model changes, so they were fully automated changes where services had APIs added or documentation updated. You could look at the changelog for one of the SDKs in another language to see exactly which services changed in that time period if you like. For example, in Go V2, anything under "Module Highlights" would be equivalent changes: https://github.com/aws/aws-sdk-go-v2/releases
